src/deps/uws.zig

```diff
@@ -2554,6 +2554,9 @@ pub const UVLoop = extern struct {
         this.uv_loop.dec();
     }
 
+    pub const ref = inc;
```

Comment:
is this for some sort of anytype interface magic thing? maybe we should just rename inc and dec to ref and unref